sdcard.c

```diff
@@ -6,6 +6,7 @@
 #include "sdcard.h"
 
 static int cmd_receive_counter;
+FILE *sdcard_file = NULL;
```

Comment:
If someone could take a look at the sdcard_file declarations/imports ... To be honest, I am slightly unsure about the importing (I really should refresh my C knowledge). Reason I am doing it that i got "asm-ld: error: duplicate symbol: sdcard_file" error when compiling with latest WebAssembly llvm.

Reply:
Your patch is correct. `extern`ed variables should go in the headers.